xmind/tests/test_markerref_markerrefselement.py

```diff
@@ -0,0 +1,28 @@
+from xmind.core.markerref import MarkerRefsElement
+from xmind.core.const import TAG_MARKERREFS
+import logging
+from . import base
+from unittest.mock import patch
+
+
+class MarkerRefsElementTest(base.Base):
+    """MarkerRefsElementTest"""
+    LOGGER = logging.getLogger('MarkerRefsElementTest')
+
+    def test_init_without_parameters(self):
+        """test that object of the class could be created without parameters and it has correct static attribute"""
+        with patch('xmind.core.mixin.WorkbookMixinElement.__init__') as WorkbookMixinElementMock:
+            _test_object = MarkerRefsElement()
+            WorkbookMixinElementMock.assert_called_with(None, None)
+
+            MarkerRefsElement('test')
+            WorkbookMixinElementMock.assert_called_with('test', None)
+
+            MarkerRefsElement('test', 2)
+            WorkbookMixinElementMock.assert_called_with('test', 2)
+
+            with self.assertRaises(Exception):
+                MarkerRefsElement('test', 2, 3)
+                WorkbookMixinElementMock.assert_called_with('test', 2, 3)
```

Comment:
This clause will be not called and checked because Exception comes before ```__init__``` gets to the call ```super().__init__```

If you are doing many test cases in one test I would recommend you to add check in the bottom of this test for how many times you expect WorkbookMixinElementMock to be called.
Like ```self.assertEqual(WorkbookMixinElementMock.call_count, 3)```

Reply:
fixed